components/top-nav/src/atoms/menu-button/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import { MEDIA_QUERIES } from '@govuk-react/constants';
+
+const Label = styled('label')({
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    display: 'none',
+  },
+});
+
+const Input = styled('input')({
+  position: 'absolute',
+  top: '-999em',
+  // This is reliant on HTML structure of parent component.
```

Comment:
Edit: I would really like it if we could support React Native and we can look at improving things like this when we get the time to.